packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -40,12 +41,78 @@ const sfdxCoreExports = vscode.extensions.getExtension(
   'salesforce.salesforcedx-vscode-core'
 )!.exports;
 const { channelService, SfdxCommandlet, notificationService } = sfdxCoreExports;
+const sfdxDeviceListCommand = 'force:lightning:local:device:list';
 const sfdxMobilePreviewCommand = 'force:lightning:lwc:preview';
 const rememberDeviceKey = 'preview.rememberDevice';
 const logLevelKey = 'preview.logLevel';
 const defaultLogLevel = 'warn';
 const androidSuccessString = 'Launching... Opening Browser';
 
+const iOSPickedDevice: vscode.QuickPickItem = {
+  label: 'iPhone 8',
+  detail: 'iOS 13.3'
+};
+const iOSDeviceListJson = `
+  {
+    "status":0,
+    "result":[
+      {
+          "name":"iPhone 8",
+          "udid":"6CC16032-2671-4BD2-8FF1-0E314945010C",
+          "state":"Shutdown",
+          "runtimeId":"iOS 13.3",
+          "isAvailable":true
+      },
+      {
+          "name":"LWCSimulator",
+          "udid":"09D522C8-DC85-4259-AD15-15D36672D2EA",
+          "state":"Shutdown",
+          "runtimeId":"iOS 13.3",
+          "isAvailable":true
+      }
+    ]
+  }
+`;
+
+const androidPickedDevice: vscode.QuickPickItem = {
+  label: 'emu2',
+  detail: 'Default Android System Image'
+};
+const androidDeviceListJson = `
+  {
+    "status":0,
+    "result":[
+      {
+          "name":"emu2",
+          "displayName":"emu2",
+          "deviceName":"pixel",
+          "path":"/Users/maliroteh/.android/avd/emu2.avd",
+          "target":"Default Android System Image",
+          "api":"API 29"
+      },
+      {
+          "name":"Pixel_API_29",
+          "displayName":"Pixel API 29",
+          "deviceName":"pixel",
+          "path":"/Users/maliroteh/.android/avd/Pixel_API_29.avd",
+          "target":"Google APIs",
+          "api":"API 29"
+      }
+    ]
+  }
+`;
+
+const createDeviceLabelText = nls.localize(
+  'force_lightning_lwc_preview_create_virtual_device_label'
+);
+const createDeviceDetailText = nls.localize(
+  'force_lightning_lwc_preview_create_virtual_device_detail'
+);
+const createNewDeviceOption: vscode.QuickPickItem = {
+  label: createDeviceLabelText,
+  detail: createDeviceDetailText
+};
+
```

Comment:
can you move this to doNewDeviceQuickPickTest? this doesn't need to be in global scope

Reply:
sure